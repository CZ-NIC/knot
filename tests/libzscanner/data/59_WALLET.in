$ORIGIN	.
$TTL	1

; The WALLET is the same as the TXT, so there are the differences and basics only.

; OK
@	WALLET	"" "test1" "\255" test2		; Array of text strings
@	WALLET	\# 1 00				; Hexadecimal rdata
@	TYPE262	\# 1 00				; TYPE + Hexadecimal rdata
@	TYPE262	""				; TYPE
@	wallet	""				; Type in lower-case

; KO
@	WALLET
