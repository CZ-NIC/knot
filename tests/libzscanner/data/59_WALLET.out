OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0106
RDATA=0005746573743101FF057465737432
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0106
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0106
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0106
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0106
RDATA=00
------
WARNG=ZS_BAD_RDATA
------
